<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_login">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>621</width>
          <height>35</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_36">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_35">
           <item>
            <widget class="QLabel" name="label_ID">
             <property name="text">
              <string>User ID: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_ID"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_34">
           <item>
            <widget class="QLabel" name="label_Password">
             <property name="text">
              <string>Password: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_password"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SignIn">
           <property name="text">
            <string>Sign In</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SignUp">
           <property name="text">
            <string>Sign Up</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_offer">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tab_Offer">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_Search">
           <attribute name="title">
            <string>Search</string>
           </attribute>
           <widget class="QGroupBox" name="group_Bid">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>370</y>
              <width>611</width>
              <height>221</height>
             </rect>
            </property>
            <property name="title">
             <string>Place bid</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>441</width>
               <height>181</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_Bargin">
                  <property name="text">
                   <string>Bargain:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="line_Bargain"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_Message">
                  <property name="text">
                   <string>Message: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="messageBox"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_2">
             <property name="geometry">
              <rect>
               <x>480</x>
               <y>30</y>
               <width>115</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="push_Bid">
                <property name="text">
                 <string>Bid</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="push_Buy">
                <property name="text">
                 <string>Buy it now!</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QGroupBox" name="group_Find">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>621</width>
              <height>201</height>
             </rect>
            </property>
            <property name="title">
             <string>Find offer</string>
            </property>
            <widget class="QWidget" name="layoutWidget_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>601</width>
               <height>171</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout"/>
                </item>
                <item>
                 <widget class="QComboBox" name="combo_Type">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="line_Search"/>
                </item>
                <item>
                 <widget class="QPushButton" name="push_Search">
                  <property name="text">
                   <string>Search</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>true</bool>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListView" name="list_Result"/>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QGroupBox" name="group_Info">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>210</y>
              <width>621</width>
              <height>151</height>
             </rect>
            </property>
            <property name="title">
             <string>Offer Information</string>
            </property>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>601</width>
               <height>111</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_Discription">
                  <property name="text">
                   <string>Description: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="line_Discription"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_Price">
                    <property name="text">
                     <string>Price: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="line_Price"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_Poster">
                    <property name="text">
                     <string>Poster: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="line_Poster"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_37">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_Barter">
                    <property name="text">
                     <string>Barter: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="line_Barter"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="label_Time">
                      <property name="text">
                       <string>Time: </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="line_Time">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_Post">
           <attribute name="title">
            <string>Post</string>
           </attribute>
           <widget class="QPushButton" name="push_Send">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>360</y>
              <width>115</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>60</y>
              <width>241</width>
              <height>50</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="label_Discription2">
               <property name="text">
                <string>Description: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_Discription2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>160</y>
              <width>194</width>
              <height>30</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QCheckBox" name="check_Cash">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_Cash">
                 <property name="text">
                  <string>Cash: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_Cash"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>200</y>
              <width>203</width>
              <height>30</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QCheckBox" name="check_Barter">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_27">
               <item>
                <widget class="QLabel" name="label_Barter2">
                 <property name="text">
                  <string>Barter: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_Barter2"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>183</x>
              <y>33</y>
              <width>171</width>
              <height>23</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <item>
              <widget class="QLabel" name="label_Item2">
               <property name="text">
                <string>Item: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_Item2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>120</y>
              <width>184</width>
              <height>23</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <item>
              <widget class="QLabel" name="label_Poster2">
               <property name="text">
                <string>Poster: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_Poster2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>240</y>
              <width>252</width>
              <height>96</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="label_Path">
                 <property name="text">
                  <string>Path: (Choose your preview picture)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_Path"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>58</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_23">
                 <item>
                  <widget class="QPushButton" name="push_Upload">
                   <property name="text">
                    <string>Upload</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="push_Browser">
                   <property name="text">
                    <string>Browser</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_View">
           <attribute name="title">
            <string>View</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget_6">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>460</y>
              <width>441</width>
              <height>36</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_33">
             <item>
              <widget class="QPushButton" name="push_Save">
               <property name="text">
                <string>Save Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="push_Delete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>230</y>
              <width>611</width>
              <height>221</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Selling:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="table_Selling"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>611</width>
              <height>221</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Buying:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="table_Buying"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
